activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -49,6 +49,9 @@ def key
       read_env_key || read_key_file || handle_missing_key
     end
 
+    # Read and return decrypted content
+    #
+    # Raise <tt>MissingContentError</tt> if unable to decrypt content
```

Comment:
This method can raise other errors as well.  For example, if `raise_if_missing_key` is true and the key is missing then the `key.nil?` call will raise `MissingKeyError`.  Plus any errors due to an incorrect key or wrong file contents.

Also, we can link `MissingContentError` to its own API documentation by omitting `<tt>`.

Reply:
Thanks for pointing that out.  I'll add that and any additional errors I see in `EncryptedFile` to the documentation. 
I also noticed that `decrypt` can raise `ActiveSupport::MessageEncryptor::InvalidMessage`, is it customary to also include that as part of documentation?  Thanks.

Reply:
Yes, let's include `ActiveSupport::MessageEncryptor::InvalidMessage` too.